quickref/qsharp-quick-reference.tex

```diff
@@ -165,22 +175,25 @@
     Call adjoint \newline operation & \texttt{Adjoint \emph{Name}(\emph{parameters})} \\
     Call controlled \newline operation & \texttt{Controlled \emph{Name}(\emph{controlQubits}, \emph{parameters})} \\
   \end{keysref}
-
   \begin{keysref}{Symbols and Variables}
     Declare immutable \newline symbol & \texttt{let \emph{name} = \emph{value}} \\
     Declare mutable \newline symbol (variable)  & \texttt{mutable \emph{name} = \emph{initialValue}} \\
     Update mutable \newline symbol (variable)   & \texttt{set \emph{name} = \emph{newValue}} \\
+    Apply-and-Reassign & \texttt{mutable \emph{name} = \emph{initialValue}} \newline
+    \texttt{for (\emph{index} in \emph{range}) \{ } \newline
+    \texttt{ set counter += \emph{index};} \texttt{\}} \\ 
   \end{keysref}
 
-  \columnbreak%\
-
   \begin{keysref}{Arrays}
```

Comment:
Arrays section needs syntax for array element assignment, per https://docs.microsoft.com/en-us/quantum/language/statements#update-and-reassign-statement